dipy/denoise/noise_estimate.py

```diff
@@ -278,10 +278,11 @@ def estimate_sigma(arr, disable_background_masking=False, N=1):
         If True, uses all voxels for the estimation, otherwise, only non-zeros
         voxels are used. Useful if the background is masked by the scanner.
 
-    N : int, default 1
+    N : int, default 0
         Number of coils of the receiver array. Use N = 1 in case of a SENSE
         reconstruction (Philips scanners) or the number of coils for a GRAPPA
-        reconstruction (Siemens and GE). See [1] for more information.
+        reconstruction (Siemens and GE). Use 0 to disable the correction factor,
+        as for example is the noise is Gaussian distributed. See [1] for more information.
```

Comment:
=> "Use 0 to disable the correction factor, for example if the noise is Gaussian distributed"